lib/msf/core/exploit/remote/kerberos/ticket.rb

```diff
@@ -150,6 +150,35 @@ def create_ccache_principal(principle, realm)
                                                                            components: principle.name_string,
                                                                            realm: realm)
           end
+
+          def ccache?(header)
+            header[0..1] == "\x05\x04"
+          end
+
+          def kirbi?(header)
+            header[0] == "\x76"
+          end
+
+          def print_contents(path, key: nil)
```

Comment:
If this is a mixin; It'll probably cause namespace issues at some point


Maybe this?

```suggestion
          def print_ccache_contents(path, key: nil)
```

(With yard docs added)

Reply:
hmm this actually calls another function called `print_ccache_contents` this one is here to figure out if it's in kirbi or ccaceh format convert if necessary and pass it on
I'll try to come up with another name